aws-rds-dbinstance/aws-rds-dbinstance.json

```diff
@@ -448,11 +457,27 @@
     },
     "read": {
       "permissions": [
+        "ec2:DescribeVpcAttribute",
+        "ec2:DescribeSecurityGroups",
+        "ec2:DescribeInternetGateways",
+        "ec2:DescribeAvailabilityZones",
+        "ec2:DescribeVpcs",
+        "ec2:DescribeAccountAttributes",
+        "ec2:DescribeSubnets",
         "rds:DescribeDBInstances"
```

Comment:
What about use rds::Describe* instead?

ref:https://aws.amazon.com/premiumsupport/knowledge-center/rds-iam-least-privileges/

Reply:
Same sentiment about using wildcards as in the comment below. I would avoid using it as far as the size of the permissions is reasonable.